operator/helper/tls.go

```diff
@@ -0,0 +1,39 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helper
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"go.opentelemetry.io/collector/config/configtls"
+)
+
+type TLSServerConfig struct {
+	*configtls.TLSServerSetting
```

Comment:
Should this have `mapstructure` / `json` / `yaml` tags?

Reply:
yeah, it's should be, let me add it